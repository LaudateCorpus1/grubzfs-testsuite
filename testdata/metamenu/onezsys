rpool/ROOT/ubuntu_1234	yes	288749dd0dc84c4cb710a7bbe83ac512	Ubuntu 19.04	1556788019	/dev/vda2	/BOOT/ubuntu_1234@/initrd.img-5.0.0-13-generic	/BOOT/ubuntu_1234@/vmlinuz-5.0.0-13-generic	/BOOT/ubuntu_1234@/vmlinuz-5.0.0-13-generic
