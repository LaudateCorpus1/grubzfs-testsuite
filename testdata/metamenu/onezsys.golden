288749dd0dc84c4cb710a7bbe83ac512	yes	main	Ubuntu 19.04	rpool/ROOT/ubuntu_1234	/dev/vda2	/BOOT/ubuntu_1234@/initrd.img-5.0.0-13-generic	/BOOT/ubuntu_1234@/vmlinuz-5.0.0-13-generic
288749dd0dc84c4cb710a7bbe83ac512	yes	advanced	Ubuntu 19.04	rpool/ROOT/ubuntu_1234	/dev/vda2	/BOOT/ubuntu_1234@/initrd.img-5.0.0-13-generic	/BOOT/ubuntu_1234@/vmlinuz-5.0.0-13-generic	true
